lib/tasks/demo_data_for_development.rake

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'webmock'
```

Comment:
Just curious: why did this line get added?

Reply:
It will throw "Undefined Webmock" or "Webmock::some_method" error without it. I'm actually not sure if this is the best place to add it. It used to work just fine.

Reply:
Ah, that's probably something to do with the Mailbluster tests. I'll look into it.